x-pack/legacy/plugins/infra/public/pages/logs/analysis/page_setup_content.tsx

```diff
@@ -132,21 +126,17 @@ export const AnalysisSetupContent: React.FunctionComponent<AnalysisSetupContentP
                 label={startTimeLabel}
               >
                 <EuiFlexGroup gutterSize="s">
-                  <EuiDatePicker
-                    showTimeSelect
-                    selected={startTime}
-                    onChange={setStartTime}
-                    placeholder={startTimeDefaultDescription}
-                    maxDate={now}
-                  />
-                  {startTime && (
-                    <EuiButtonIcon
-                      color="danger"
-                      iconType="cross"
-                      aria-label={clearStartTimeLabel}
-                      onClick={() => setStartTime(null)}
+                  <EuiFormControlLayout
```

Comment:
This is cool! My change to EUI already landed so as soon as I can get that merged, we should just move this onClick function to be `onClear` for `EuiDatePicker` since I think that's a little ... ahem ... clearer :P